src/python/TaskWorker/Actions/DBSDataDiscovery.py

```diff
@@ -269,6 +271,9 @@ def executeInternal(self, *args, **kwargs):
     # will user service cert as defined for TW
     config.TaskWorker.cmscert = os.environ["X509_USER_CERT"]
     config.TaskWorker.cmskey = os.environ["X509_USER_KEY"]
+    config.TaskWorker.envForCMSWEB = newX509env(X509_USER_CERT= config.TaskWorker.cmscert,
```

Comment:
Isn't this redundant given the two lines above?

Reply:
Not at all.
This only "prepares and stores" a context manager which can be used at any time in the code to make sure that some opearions are done with the proper environment variables for talking with CMSWEB via
```
with config.TaskWorker.envForCMSWEB :
    do something
```